iphone/DatePicker/DatePicker/Cells/SelectedSingleCell.swift

```diff
@@ -7,6 +7,9 @@ class SelectedSingleCell: Cell {
     contentView.addSubview(selectedBgView)
     selectedBgView.alignToSuperview(UIEdgeInsets(top: 4, left: 4, bottom: -4, right: -4))
     selectedBgView.layer.cornerRadius = 8
```

Comment:
Can this setter be overrided to automatically set `layer.cornerCurve = .continuous` on iOS 13+? Or is it a bad idea?